source/tools/MatchBin.h

```diff
@@ -12,6 +12,11 @@
 #ifndef EMP_MATCH_BIN_H
 #define EMP_MATCH_BIN_H
 
+// take care of preprocessored filename
```

Comment:
add comment about how to set the default filename with -D

Reply:
Implemented in 93e96aa.